test-integ/go.mod

```diff
@@ -90,7 +90,7 @@ require (
 	github.com/googleapis/gax-go/v2 v2.11.0 // indirect
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0 // indirect
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3 // indirect
-	github.com/hashicorp/consul v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
This is not really related but I decided to add it, as it should not matter and my IDE annoyingly update this every time I make a change and I'm pretty sure I'm not the only one, I've seen it in others setup.